modules/hosted-control-planes-concepts-personas.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * architecture/control-plane.adoc
+// * hosted-control-planes/index.adoc
+
+
+:_content-type: CONCEPT
+[id="hosted-control-planes-concepts-personas_{context}"]
+= Glossary of common concepts and personas for hosted control planes
+
+When you use hosted control planes for {product-title}, it is important to understand its key concepts and the personas that are involved.
+
+[id="hosted-control-planes-concepts_{context}"]
+== Concepts
+
+hosted cluster:: An {product-title} API endpoint for the tenant cluster that is managed by the HyperShift Operator.
+
+hosted cluster infrastructure:: Network, compute, and storage resources that exist in the tenant or end-user cloud account.
+
+hosting cluster:: See _management cluster_.
+
+hosted control plane:: An {product-title} control plane that runs on the management cluster, which is exposed by the API endpoint of a hosted cluster. The components of a control plane include etcd, an API server, the Kubernetes controller manager, and a VPN.
+
+management cluster:: An {product-title} cluster where the HyperShift Operator is deployed and where the control planes for tenant clusters are hosted. In some contexts, the management cluster is referred to as the _hosting cluster_.
+
+management cluster infrastructure:: Network, compute, and storage resources of the management cluster.
+
+[id="hosted-control-planes-personas_{context}"]
+== Personas
+
+cluster instance administrator:: The role that is assumed by the user who is the equivalent to the administrator in stand-alone {product-title}. This user has the cluster-admin role in the provisioned cluster, but might not have power over when or how the cluster is updated or configured. This user might have read-only access to see some configuration projected into the cluster.
```

Comment:
* "standalone" has no hyphen (per SSG's [standalone entry](https://redhat-documentation.github.io/supplementary-style-guide/#standalone)) (here and in the next definition too)
* cluster-admin in backticks

```suggestion
cluster instance administrator:: The role that is assumed by the user who is the equivalent to the administrator in standalone {product-title}. This user has the `cluster-admin` role in the provisioned cluster, but might not have power over when or how the cluster is updated or configured. This user might have read-only access to see some configuration projected into the cluster.
```